br/pkg/restore/db.go

```diff
@@ -190,8 +190,8 @@ func (db *DB) CreateTable(ctx context.Context, table *metautil.Table, ddlTables
 				zap.Error(err))
 			return errors.Trace(err)
 		}
-	// only table exists in ddlJobs during incremental restoration should do alter after creation.
-	case ddlTables[UniqueTableName{table.DB.Name.String(), table.Info.Name.String()}]:
+	// only table exists in restored cluster during incremental restoration should do alter after creation.
+	case toBeCorrectedTables[UniqueTableName{table.DB.Name.String(), table.Info.Name.String()}]:
```

Comment:
I remember the name has changed🤔

Reply:
I also remember I changed the name. But I checked the origin PR, I only changed the `Client.rebasedTablesMap`, not changed this param. I get in trouble how to change codes here. I think maybe changing the name in another PR is better?